--- vendor/sony/lagan/proprietary/Android.mk	2013-11-04 18:14:36.397161824 +0100
+++ vendor/sony/lagan/proprietary/Android.mk	2013-11-04 18:14:51.337593789 +0100
@@ -77,16 +77,6 @@
 include $(BUILD_PREBUILT)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := qrngd
-LOCAL_MODULE_OWNER := Sony Mobile
-LOCAL_SRC_FILES := bin/qrngd
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_SUFFIX := 
-LOCAL_MODULE_CLASS := EXECUTABLES
-LOCAL_MODULE_PATH := $(TARGET_OUT)/bin
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
 LOCAL_MODULE := qseecomd
 LOCAL_MODULE_OWNER := Sony Mobile
 LOCAL_SRC_FILES := bin/qseecomd
@@ -1311,16 +1301,6 @@
 include $(BUILD_PREBUILT)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := liblights-core
-LOCAL_MODULE_OWNER := Sony Mobile
-LOCAL_SRC_FILES := lib/liblights-core.so
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_SUFFIX := .so
-LOCAL_MODULE_CLASS := SHARED_LIBRARIES
-LOCAL_MODULE_PATH := $(TARGET_OUT)/lib
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
 LOCAL_MODULE := libllvm-a3xx
 LOCAL_MODULE_OWNER := Sony Mobile
 LOCAL_SRC_FILES := lib/libllvm-a3xx.so
@@ -1691,16 +1671,6 @@
 include $(BUILD_PREBUILT)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := libqservice
-LOCAL_MODULE_OWNER := Sony Mobile
-LOCAL_SRC_FILES := lib/libqservice.so
-LOCAL_MODULE_TAGS := optional
-LOCAL_MODULE_SUFFIX := .so
-LOCAL_MODULE_CLASS := SHARED_LIBRARIES
-LOCAL_MODULE_PATH := $(TARGET_OUT)/lib
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
 LOCAL_MODULE := libquipc_os_api
 LOCAL_MODULE_OWNER := Sony Mobile
 LOCAL_SRC_FILES := lib/libquipc_os_api.so
@@ -1880,4 +1850,4 @@
 LOCAL_MODULE_PATH := $(TARGET_ROOT_OUT)/sbin
 include $(BUILD_PREBUILT)
 
-endif
\ No newline at end of file
+endif
